tests/foreman/virtwho/ui/test_esx_sca.py

```diff
@@ -0,0 +1,710 @@
+"""Test class for Virtwho Configure UI
+
+:Requirement: Virt-whoConfigurePlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Virt-whoConfigurePlugin
+
+:Team: Phoenix
+
+:TestType: Functional
+
+:Upstream: No
+"""
+from datetime import datetime
+
+import pytest
+from airgun.session import Session
+from fauxfactory import gen_string
+
+from robottelo.config import settings
+from robottelo.utils.datafactory import valid_emails_list
+from robottelo.utils.virtwho import add_configure_option
+from robottelo.utils.virtwho import delete_configure_option
+from robottelo.utils.virtwho import deploy_configure_by_command
+from robottelo.utils.virtwho import deploy_configure_by_command_check
+from robottelo.utils.virtwho import deploy_configure_by_script
+from robottelo.utils.virtwho import ETC_VIRTWHO_CONFIG
+from robottelo.utils.virtwho import get_configure_command
+from robottelo.utils.virtwho import get_configure_file
+from robottelo.utils.virtwho import get_configure_id
+from robottelo.utils.virtwho import get_configure_option
+from robottelo.utils.virtwho import get_guest_info
+from robottelo.utils.virtwho import get_virtwho_status
+from robottelo.utils.virtwho import restart_virtwho_service
+from robottelo.utils.virtwho import update_configure_option
+
+
+@pytest.fixture()
```

Comment:
non-blocking:
I would suggest that we can put all the `form_data `methods creating `virt_who.py` in pytest_fixtures and it would be great if we can parameterize this one for hyperv/esx/kubevirt/libvirt as I see a good amount of analogy in the way they are defined.

Reply:
@adarshdubey-star Have already put the session create and delete in fixture